client/profile-wizard/steps/store-details/index.js

```diff
@@ -206,7 +206,7 @@ export class StoreDetails extends Component {
 		const validateAddress = getStoreAddressValidator( locale );
 		const errors = validateAddress( values );
 
-		if ( ! isEmail( values.storeEmail ) ) {
+		if ( values.storeEmail && ! isEmail( values.storeEmail ) ) {
```

Comment:
Should we check this with `values.isAgreeMarketing`? Looks like the email field is only optional when the`isAgreeMarketing` is `false`.

https://github.com/woocommerce/woocommerce-admin/blob/dbd62e4f0671707833dceb26238d44b1f12d7b26/client/profile-wizard/steps/store-details/index.js#L339-L347

Reply:
Hmm, It seems that it also works because we have `required={ values.isAgreeMarketing }` to prevent users go to the next step. It's probably fine with `values.storeEmail` check. Please let me know your thoughts.

Reply:
Okay, that makes sense!